app/models/dynamic_reports.rb

```diff
@@ -888,7 +888,8 @@ class  ServiceStatusReport
     filter(:created_at,
            :date,
            :range => true,
-           :default => proc { [Service.all.select(:created_at).order("created_at ASC").first.created_at, Date.today]},
+           :default => proc { [Service.all.select(:created_at).order("created_at ASC").first.blank? ? Date.today : Service.all.select(:created_at).order("created_at ASC").first.created_at,
```

Comment:
Line is too long. [188/80]<br>Avoid using `{...}` for multi-line blocks.<br>Use the new Ruby 1.9 hash syntax.<br>Block body expression is on the same line as the block start.